vars/Debian_bookworm.yml

```diff
@@ -14,10 +14,18 @@ python2_package_names:
   # This package provides the /usr/bin/python symlink
   # - python2-minimal
 
+# The Python 2 development package names
+# python2_development_package_names:
+#   - python2-dev
```

Comment:
This is inconsistent with the section above it, which has `python2_package_names` uncommented.  Not sure which way you meant to go here, just noting that they should be consistent.

Reply:
Please see commit 984cdbd.

Reply:
Excellent 👍